src/NHSD.BuyingCatalogue.Identity.Api/ViewModels/Account/LogoutViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace NHSD.BuyingCatalogue.Identity.Api.ViewModels.Account
+{
+    public sealed class LogoutViewModel
+    {
+        public string LogoutId { get; set; }
```

Comment:
Worth adding a `Required` attribute here as above?

Reply:
Deleted the class, no longer required.